msf/printerscript.go

```diff
@@ -6,7 +6,7 @@ import (
 )
 
 func getTimeEstimate(command gcode.Command, state *State) float32 {
-	if command.IsLinearMove() {
+	if command.IsLinearOrArcMove() {
```

Comment:
I think arc moves should use a different time estimation calculation than linear moves – this is because arc moves don't involve moving in a straight line from the start point to the end point, but rather involve moving along the arc (where the length is the arc length, rather than the straight-line distance). Something like the following:

```go
func estimateMoveTime(arcLength, feedrate float32) float32 {
	mmPerS := feedrate / 60
	return arcLength / mmPerS
}
```